scripts/find-internal-packages.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+source $OKTA_HOME/$REPO/scripts/setup.sh
+
+if ! NODE_TLS_REJECT_UNAUTHORIZED=0 yarn find-internal-packages; then
```

Comment:
why do we need to turn off ssl for this?

Reply:
Without this npm-check returns error `Registry error self signed certificate in certificate chain` for all npm packages.
Probably related to this: https://github.com/okta/okta-signin-widget/blob/master/scripts/setup.sh#L12-L13

Reply:
Do we need the full feature set of `npm-check`? Maybe `npm outdated` can work for us?

https://github.com/dylang/npm-check/issues/11#issuecomment-294296207

Reply:
`npm outdated` just checks new versions of installed packages.
`npm-check` can analyse sources and detect packages that are referenced in sources but not present in NPM registry - this feature is very useful to detect internal packages, I haven't found alternatives to it